src/main/java/de/retest/recheck/ui/descriptors/AttributeUtil.java

```diff
@@ -52,6 +54,10 @@ public static Rectangle getAbsoluteOutline( final IdentifyingAttributes attribut
 	}
 
 	private static Rectangle getActualOutline( final ElementDifference difference, final String type ) {
+		if (!(difference.getIdentifyingAttributesDifference() instanceof IdentifyingAttributesDifference)) {
```

Comment:
`difference.getIdentifyingAttributesDifference()` is called twice, maybe extract the value earlier without cast?

Reply:
I think this is really just a getter, extracting this into a variable will bloat the code and make it less readable.